<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<comment> Copyright (C) 2019 - 2021 by Pedro Mendes, Rector and Visitors of the </comment>
<comment> University of Virginia, University of Heidelberg, and University </comment>
<comment> of Connecticut School of Medicine. </comment>
<comment> All rights reserved. </comment>

<comment> Copyright (C) 2017 - 2018 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and University of </comment>
<comment> of Connecticut School of Medicine. </comment>
<comment> All rights reserved. </comment>

<comment> Copyright (C) 2012 - 2016 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., University of Heidelberg, and The University </comment>
<comment> of Manchester. </comment>
<comment> All rights reserved. </comment>

 <comment> All rights reserved. </comment>
 <class>CQParameterOverviewWidget</class>
 <widget class="CopasiWidget" name="CQParameterOverviewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>190</height>
   </rect>
  </property>
  <property name="caption" stdset="0">
   <string>Model Parameter Set</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="mpHeaderWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="mpLblHeader">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Lucida Grande'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:16px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:x-large; font-weight:600;&quot;&gt;Model Parameters&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mpBtnSaveToFile">
        <property name="text">
         <string>Save to File</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="mpTreeView">
     <property name="editTriggers">
      <set>QAbstractItemView::AllEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
     <property name="rootIsDecorated">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mpBtnWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="mpBtnCommit">
        <property name="text">
         <string>Commit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mpBtnRevert">
        <property name="text">
         <string>Revert</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="mpBtnSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>18</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="mpBtnSaveAs">
        <property name="text">
         <string>Store as Parameter Set</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CopasiWidget</class>
   <extends>QWidget</extends>
   <header>copasi/UI/copasiWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mpBtnSaveToFile</tabstop>
  <tabstop>mpTreeView</tabstop>
  <tabstop>mpBtnCommit</tabstop>
  <tabstop>mpBtnRevert</tabstop>
  <tabstop>mpBtnSaveAs</tabstop>
 </tabstops>
 <includes>
  <include location="global">vector</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>mpBtnCommit</sender>
   <signal>clicked()</signal>
   <receiver>CQParameterOverviewWidget</receiver>
   <slot>slotBtnCommit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnRevert</sender>
   <signal>clicked()</signal>
   <receiver>CQParameterOverviewWidget</receiver>
   <slot>slotBtnRevert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnSaveToFile</sender>
   <signal>clicked()</signal>
   <receiver>CQParameterOverviewWidget</receiver>
   <slot>slotBtnSaveToFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnSaveAs</sender>
   <signal>clicked()</signal>
   <receiver>CQParameterOverviewWidget</receiver>
   <slot>slotBtnSaveAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
  <slots>
    <slot>slotBtnRevert()</slot>
    <slot>slotBtnCommit()</slot>
    <slot>slotBtnNew()</slot>
    <slot>slotBtnCopy()</slot>
    <slot>slotBtnDelete()</slot>
    <slot>slotBtnSaveToFile()</slot>
    <slot>slotBtnSaveAs()</slot>
    <slot>saveParameterSet(CModelParameterSet *)</slot>
    <slot>slotOpenEditor(QModelIndex)</slot>
    <slot>slotCloseEditor(QModelIndex)</slot>
    <slot>slotResolve(QModelIndex)</slot>
    <slot>slotFilterChanged()</slot>
  </slots>
</ui>
